users/views.py

```diff
@@ -1,9 +1,27 @@
 from rest_framework import generics
-from rest_framework.permissions import AllowAny
+from rest_framework.permissions import AllowAny, IsAuthenticated
 
-from users.serializers import SignupSerializer
+from users.models import User
+from users.serializers import SignupSerializer, UpdateSerializer, UserPrivateSerializer
 
 
 class SignupView(generics.CreateAPIView):
     permission_classes = (AllowAny,)
     serializer_class = SignupSerializer
+
+
+class GetView(generics.RetrieveAPIView):
```

Comment:
We have [RetrieveUpdateAPIView](https://www.django-rest-framework.org/api-guide/generic-views/#retrieveupdateapiview)

Reply:
Changed to RetrieveUpdateAPIView